### ISSUE TITLE
Minnows question

### ISSUE THREAD

**Issue by darkjeffrey** [wikiworm/InventoryValue#20]
Would it be possible to assign 40 minnows the price on one raw shark?

**Comment by wikiworm** [comment]
Good idea. I'll get this added and close this issue once the change is published to pluginhub.
